Zend/tests/unexpected_ref_bug.phpt

```diff
@@ -9,12 +9,11 @@ class Test {
         return ",";
     }
 }
-$my_var = str_repeat("A",64);
-$data = call_user_func_array("explode",array(new Test(), &$my_var));
-$my_var=array(1,2,3);
-$data = call_user_func_array("implode",array(&$my_var, new Test()));
+$my_var = str_repeat("A", 64);
+$data = call_user_func_array("explode", array(new Test(), &$my_var));
+$my_var = str_repeat("A", 64);
+$data = call_user_func_array("implode", array(&$my_var, []));
```

Comment:
This no longer tests what it's supposed to. It should use some other function that takes a string as second arg. Like `call_user_func_array("str_replace", array(&$my_var, new Test(), "foo"))` or something.

Reply:
Thanks for the suggestion!